sandbox-backend/src/main/resources/db/migration/V0000000__initial.sql

```diff
@@ -7,12 +7,20 @@ CREATE TABLE IF NOT EXISTS `DEMO_TABLE`
 ) ENGINE = InnoDB
   DEFAULT CHARSET = UTF8;
 
+CREATE TABLE country
```

Comment:
Please create a new migration script for these changes.

Reply:
done